bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/sound-controller.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="sound-controller">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Sound Controller</label>
+		<description>Controller for speakers</description>
+
+		<channels>
+			<channel id="battery-level" typeId="percent">
```

Comment:
You could use the system state channel `system.battery-level`
also applies to other `Things`

Reply:
Done